manager/integration/tests/test_metric.py

```
@@ -1,12 +1,14 @@
 import pytest
 import requests
 import time
+import backupstore
```

Ruff

`backupstore` imported but unused

Remove unused import: `backupstore`

(F401)